modules/loki-rbac-rules-permissions.adoc

```diff
@@ -0,0 +1,60 @@
+// Module included in the following assemblies:
+//
+// * logging/logging_alerts/custom-logging-alerts.adoc
+
+:_mod-docs-content-type: REFERENCE
+[id="loki-rbac-rules-permissions_{context}"]
+= Authorizing LokiStack rules RBAC permissions
+
+Administrators can allow users to create and manage their own alerting and recording rules by binding cluster roles to usernames.
+Cluster roles are defined as `ClusterRole` objects that contain necessary role-based access control (RBAC) permissions for users.
+
+In logging 5.8 and later, the following cluster roles for alerting and recording rules are available for LokiStack:
+
+[options="header"]
+|===
+|Rule name |Description
+
+|`alertingrules.loki.grafana.com-v1-admin`
+|Users with this role have administrative-level access to manage alerting rules across the entire cluster. This cluster role grants permissions to create, read, update, delete, list, and watch `AlertingRule` resources within the `loki.grafana.com/v1` API group across all namespaces in the cluster.
```

Comment:
> This cluster role grants permissions [...] across all namespaces in the cluster.

Whether a role allows something "across the whole cluster" or "just in a namespace" is decided by the binding. When a `RoleBinding` is used, then it only applies to a namespace (this is what `add-role-to-user` creates) and if a `ClusterRoleBinding` is used then it applies to all namespaces in the cluster (`add-cluster-role-to-user`).

Reply:
I wasn't sure about this, because my thinking then was, what's the difference between `edit` vs `admin`? Is it just stuff like `view`/`list`/`watch` that's different, or maybe something else?

Reply:
Ah, I see the comment below clarifies - thank you!